docs/content/en/docs/getting-started/_configuration/cluster_clusterNetwork.html

```diff
@@ -36,6 +36,11 @@
 hands traffic destined for that range to the Linux network stack without
 applying any SNAT.
 
+### clusterNetwork.cniConfig.cilium.cniExclusive (optional)
+Controls whether Cilium should remove other CNI configuration files.
+When true (default), Cilium removes other CNI configs; when false, it leaves them alone.
+For more information, see <a href="/docs/getting-started/optional/cni/#cni-exclusive-mode-configuration">CNI Exclusive Mode Configuration</a>.
```

Comment:
```suggestion
For more information, see <a href="/docs/getting-started/optional/cni/#cni-exclusive-mode-configuration">CNI Exclusive Mode configuration</a>.
```
We use sentence case in headings. I assume that "Exclusive Mode" is sort of a title, so we can leave that as caps.

Reply:
Makes sense. Thanks for the suggestion.